### DISCUSSION TITLE
thermal failure bug

### DISCUSSION THREAD

**Discussion by grrmisfit** [mriscoc/Ender3V2S1#60] [Q&A] [answered]
as discussed here https://github.com/MarlinFirmware/Marlin/issues/23581 can you change this? my S1 has constant thermal failures at random times tho temp listed is stable

**Comment by mriscoc** [comment, marked as answer]
Thank you for your report. Seems that the problem is caused by the new feature `THERMAL_PROTECTION_VARIANCE_MONITOR`. I will disable it from the configuration files in the next release until that will be stable.

**Reply by mriscoc** [reply]
Just released a Standard 20220224 beta with that feature disabled https://github.com/mriscoc/Ender3S1/releases/tag/20220224
The Enhanced Professional version will be available soon after testing in the Telegram group.

**Reply by grrmisfit** [reply]
thank you will test now

**Reply by grrmisfit** [reply]
so far no issues, thanks again,

**Reply by mriscoc** [reply]
Thank you for your test. The Enhanced new version is currently under test, I hope to release it next week.

**Reply by Tillerman6** [reply]
Just to possibly verify this thermal issue- Was testing performance  422 + BL touch  20220220.bin download with PLA and pushing the speed up to 150%  at 200 degrees- (the slicer settings)  but the filament was designed for 220.  I started the print at 220, but while I wasn't looking the temp went back to 200 and the printer gave a two beep warning and said "nozzle temp too low" and the printer just stopped right there.  - the print was not re-start able.  My fault entirely, but what would the beta version do in this scenario?

**Reply by mriscoc** [reply]
Hi. As I wrote before, the next version will have disabled the feature: THERMAL_PROTECTION_VARIANCE_MONITOR, that version will be released the next week, currently that version is under test in our Telegram group: https://t.me/ender3v2s1firmware
